forms-flow-web/src/components/ServiceFlow/ServiceFlow.scss

```diff
@@ -21,6 +21,70 @@
   overflow-y: auto;
   overflow-x: hidden;
 }
+
+.service-task-action{
+  margin-bottom: -8px;
+  font-size: 14px;
+
+}
+
+.handle-view-details{
+  color: var(--color-primary-dkr);
+  text-decoration: none;
+}
+.word-break{
+  word-break: break-all;
+}
+
+.no-filter-img{
+  height: 150px;
+
+}
+
+.task-attribute{
+ margin-bottom: -2.5rem; 
+}
+
+.task-item-label{
+  margin-bottom: 0px;
+}
+
+.truncate-size{
+  font-size: 14px;
+}
+
+.filter-listview{
+  min-width: 700px;
+}
+
+.filter-list{
+  margin-top: 45px;
+}
+
+.create-new-filter{
+ font-size: 16px; 
+}
+.list-size{
```

Comment:
you can change class name because this class name is not understandable its behaviour example :` fs-18`